scripts/zones/Quicksand_Caves/npcs/amk_qm.lua

```diff
@@ -0,0 +1,12 @@
+-----------------------------------
```

Comment:
This is the correct naming for file here, can you update npc_list and the mission script to key on this?